questdb-rs/src/ingress/mod.rs

```diff
@@ -43,87 +48,147 @@
 //!
 //! fn main() -> Result<()> {
 //!    let mut sender = Sender::from_conf("http::addr=localhost:9000;")?;
-//!    let mut buffer = Buffer::new();
-//!    buffer
+//!    let mut buf = Buffer::new();
+//!    buf
 //!        .table("sensors")?
 //!        .symbol("id", "toronto1")?
 //!        .column_f64("temperature", 20.0)?
 //!        .column_i64("humidity", 50)?
 //!        .at(TimestampNanos::now())?;
-//!    sender.flush(&mut buffer)?;
+//!    sender.flush(&mut buf)?;
 //!    Ok(())
 //! }
 //! ```
 //!
-//! # Flushing
+//! # Unified APIs: configuration string and builder API
```

Comment:
I'm not sure "Unified APIs" means much. Maybe just "Configuration Parameters"?
The content itself is great.

Reply:
This heading covers just the single paragraph below it, which explains that there's the Builder struct whose API is identical to the configuration string.